From 088408c085e5d668c872769391787481875293a6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C4=90o=C3=A0n=20Tr=E1=BA=A7n=20C=C3=B4ng=20Danh?=
 <congdanhqx@gmail.com>
Date: Tue, 15 Sep 2020 20:51:28 +0700
Subject: [PATCH 3/3] sysprof-capture-condition: always return even if
 unreachable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix the problem with -Werror=return-type

Signed-off-by: Đoàn Trần Công Danh <congdanhqx@gmail.com>
---
 src/libsysprof-capture/sysprof-capture-condition.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libsysprof-capture/sysprof-capture-condition.c b/src/libsysprof-capture/sysprof-capture-condition.c
index 8f89157..4b829a9 100644
--- a/src/libsysprof-capture/sysprof-capture-condition.c
+++ b/src/libsysprof-capture/sysprof-capture-condition.c
@@ -269,6 +269,7 @@ sysprof_capture_condition_copy (const SysprofCaptureCondition *self)
     }
 
   sysprof_assert_not_reached ();
+  return NULL;
 }
 
 static void
-- 
2.28.0

